src/pep8_naming/checks.rs

```diff
@@ -114,12 +114,36 @@ pub fn dunder_function_name(func_def: &Stmt, scope: &Scope, name: &str) -> Optio
     None
 }
 
+fn is_lower(s: &str) -> bool {
+    let mut cased = false;
+    for c in s.chars() {
+        if c.is_uppercase() {
+            return false;
+        } else if !cased && c.is_lowercase() {
+            cased = true;
+        }
+    }
+    cased
+}
```

Comment:
Code borrowed from cpython:

https://github.com/python/cpython/blob/7108bdf27c7a460cf83c4a01dea54ae4591d8aea/Objects/bytes_methods.c#L184

Reply:
Awesome, can we add a test or two for these functions?